docs/sources/reference/components/prometheus/prometheus.exporter.snmp.md

```diff
@@ -334,6 +255,102 @@ The YAML file in this example looks like this:
     auth: public_v2
 ```
 
+## Example using the target block (deprecated)
```

Comment:
did you build the docs and checked that it's correctly rendered? The github linting is showing weird colors but it's probably a lint bug because it seems ok to me

Reply:
It displays ok on the website. But anyway, not I refactored the markdown and this is a `###` instead of a `##`